test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -87,8 +87,8 @@ class NodeWatcher {
     node.layer->setTrainable(true);
 
     for (unsigned int i = 0; i < num_weights; ++i) {
-      const nntrainer::Weight &w = node.layer->weightAt(i);
-      expected_weights.push_back(w);
+      nntrainer::Weight &w = node.layer->weightAt(i);
```

Comment:
I guess this should remain const and change signature of `weight::clone const ()`?

Reply:
done.